setup.py

```diff
@@ -17,7 +17,7 @@
 
 setup(
     name='domcheck',
-    version='1.1.4',
+    version='1.1.4.zapier',
```

Comment:
@parsenz We should use `post1` here instead of using `zapier` as it does not conform to https://www.python.org/dev/peps/pep-0440/. This means that tools like `poetry` or `pipenv` won't be able to process this library as a dependency.

Reply:
Cool! I'll update this - I was following the instructions here: https://zapier.quip.com/zJ5wAaF2844A/Deploying-Python-Packages-with-Wheels#YWSACARn2wR